tests/HealthTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Tests;
+
+use OpenRuntimes\Proxy\Health\Health;
```

Comment:
I have methods `testNode` and `testHealth`. Should I separate this into 2 files? Or do I not cover Node class well in there?

Reply:
I usually like to decouple or repeat the tests just to ensure I have good coverage and in case we need to expand some of the tests in the future even if in practice the existing coverage is good enough. Especially when it comes to unit tests.

Reply:
Separated into HealthTest and NoteTest.